packages/e2e-tests/specs/editor/various/inserter.test.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	createNewPost,
+	openGlobalBlockInserter,
+} from '@wordpress/e2e-test-utils';
+
+describe( 'Inserter', () => {
+	it( 'shows block preview when hovering over block in inserter', async () => {
+		await createNewPost();
+		await openGlobalBlockInserter();
+		await page.focus( '.editor-block-list-item-paragraph' );
+		await page.waitForSelector( '.block-editor-inserter__preview', {
```

Comment:
Mostly curious if this call also returns the same element handle like the next line. According to the documentation it might.

Reply:
Ah, nice spot! Seems to work like that indeed :+1:

(Tested to fail when a224cbf is reverted)

Reply:
Thanks for checking